aldourtusuastegui/MovieTime
# Android |Feature/profile | Show profile information getting data from Api Response and Room

### **What is solving?** 
get profile data from most Popular actor from `themoviedb` API and save it data in room.


### **Solution description**
-created feature_profile with clean architecture and MVVM.
-added room
-added hilt
-data is obtained from api
-data is obtained from local database when device has not connection to internet.
-displayed information of profile.

### **Steps to verify changes**
-Run the application or see attached video



https://github.com/aldourtusuastegui/MovieTime/assets/13209412/f0db4ef7-b13a-42bb-8e19-877aab254e78
